src/core/server/graph/loaders/Comments.ts

```diff
@@ -190,21 +190,34 @@ export default (ctx: GraphContext) => ({
     tag,
     query,
     orderBy,
-  }: QueryToCommentsArgs) =>
-    retrieveCommentConnection(ctx.mongo, ctx.tenant.id, {
-      first: defaultTo(first, 10),
-      after,
-      orderBy: defaultTo(orderBy, GQLCOMMENT_SORT.CREATED_AT_DESC),
-      filter: {
-        ...queryFilter(query),
-        ...tagFilter(tag),
-        ...sectionFilter(ctx.tenant, section),
-        // If these properties are not provided or are null, remove them from
-        // the filter because they do not exist in a nullable state on the
-        // database model.
-        ...requiredPropertyFilter({ storyID, siteID, status }),
+  }: QueryToCommentsArgs) => {
+    let story: Readonly<Story> | null = null;
+    if (storyID) {
+      story = await retrieveStory(ctx.mongo.live, ctx.tenant.id, storyID);
```

Comment:
Should use the loader here:

```suggestion
      story = await ctx.loaders.Stories.story.load(storyID)
```